controllers/mirrorpeer_controller.go

```diff
@@ -508,3 +518,39 @@ func (r *MirrorPeerReconciler) createDRClusters(ctx context.Context, mp *multicl
 	}
 	return nil
 }
+
+func (r *MirrorPeerReconciler) createRoleBindingsForSpoke(ctx context.Context, peer multiclusterv1alpha1.MirrorPeer) error {
+	var subjects []rbacv1.Subject
+	for _, pr := range peer.Spec.Items {
+		subjects = append(subjects, rbacv1.Subject{
+			Kind:     "User",
+			Name:     agent.DefaultUser(pr.ClusterName, addons.TokenExchangeName, addons.TokenExchangeName),
+			APIGroup: "rbac.authorization.k8s.io",
+		})
+
+		subjects = append(subjects, rbacv1.Subject{
+			Kind:     "Group",
+			Name:     agent.DefaultGroups(pr.ClusterName, addons.TokenExchangeName)[0],
+			APIGroup: "rbac.authorization.k8s.io",
+		})
+	}
+
+	crb := rbacv1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: spokeClusterRoleBindingName,
```

Comment:
For all mirror peer cluster set is it uses the same name?

Reply:
i am going to change this to add subjects to the same name.
Each unique subject will be in this

Reply:
ack